src/lib/api/Operator.ts

```diff
@@ -94,6 +94,27 @@ class Operator {
     }
   }
 
+  async updateOperatorMetadata(operatorId: string, signature: string, operatorMetadata: Record<string, any>) {
+    const url = `${String(config.links.SSV_API_ENDPOINT)}/operators/${operatorId}/metadata`;
+    try {
+      return (await axios.put(url, {
+        ...operatorMetadata,
+        signature,
+      })).data;
+    } catch (e) {
+      throw e;
```

Comment:
What's the sense to catch and throw it right away, you can remove try-catch block in such case

Reply:
removed